baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -22,6 +22,7 @@ import type {Callback, CallOptions, Descriptors, ClientOptions, GrpcClientOption
 
 import * as protos from '../../protos/protos';
 import jsonProtos = require('../../protos/protos.json');
+import * as crypto from 'crypto';
```

Comment:
This immediately disqualifies the library from the browser or any other non-Node.js usage. One way to solve this would be to move everything that needs `crypto` to google-gax and export it from there, and in gax, make it similar to https://github.com/googleapis/google-auth-library-nodejs/tree/main/src/crypto to use either Node.js crypto or `window.crypto.subtle`, whichever is available.

Also, I don't see any `crypto` usage in this particular file, which probably means that this import should be conditional.